erpnext_com/www/docs/user/manual/en/setting-up/chart-of-accounts-importer.md

```diff
@@ -0,0 +1,50 @@
+<!--add breadcrumbs-->
+
+# Chart Of Accounts Importer
+
+>Introduced in Version 12
+
+Chart of Accounts Importer is a great tool to import your existing chart of accounts. It allows you to create your own chart of accounts according to your requirement and import it into the system .
```

Comment:
add: If you have any existing accounts created, they will be overwritten. However, make sure the company you are selecting doesn't have a pre existing transactions otherwise you'll receive a validation error.